packages/hoprd/crates/hoprd-api/src/lib.rs

```diff
@@ -1019,30 +1054,26 @@ mod messages {
 
         let size = inbox.read().await.size(Some(tag.tag)).await;
 
-        Ok(Response::builder(200).body(json!(Size{size})).build())
+        Ok(Response::builder(200).body(json!(Size { size })).build())
     }
 
     #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
     #[serde(rename_all = "camelCase")]
     struct MessagePopRes {
         tag: u16,
         body: String,
-        received_at: u128
+        received_at: u128,
```

Comment:
Same as above, `u128` is an overkill and use `std::time::Duration`.
```suggestion
        #[serde_as(as = "serde_with::DurationMilliSeconds<u64>")]
        received_at: std::time::Duration,
```
(note you need the `#[serde_as]` before `#[derive(...)]` for this to work).

Reply:
Ok but this is not done by this PR. It was done already before.